src/community/index.md.njk

```diff
@@ -1,52 +1,171 @@
+
+[Sign up to get emails about the Design System](https://mailchi.mp/707ce8dec373/get-updated-by-email-govuk-design-system). We regularly reach out to our community to:
+- announce new events and workshops
+- conduct surveys to help guide our work
+- ask service teams to help test changes before we release them
+
+<hr class="govuk-section-break govuk-section-break--visible">
+
+## Join the conversation
+
+Here’s a few places to join the discussions that help shape the Design System.
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-one-half">
+    {%- set chatWithUsOnSlack -%}
+      <p>If you work in government, find us on the <a href="https://ukgovernmentdigital.slack.com/archives/C6DMEH5R6">#govuk-design-system Slack channel</a> to get support, give us feedback or share your work with us.</p>
+    {%- endset -%}
+
+    {{- imageCard({
+      src: "/community/images/chat-with-us-on-slack.jpg",
+      alt: "Slack conversation illustration.",
+      title: "Chat with us on Slack",
+      content: chatWithUsOnSlack
+    }) }}
+  </div>
+  <div class="govuk-grid-column-one-half">
+    {%- set discussAndGiveFeedback -%}
+      <p>Tell us your experience using our components and patterns. Look for the ‘Help improve this page’ section at the end of each page to see its issue discussion, or <a href="https://github.com/alphagov/govuk-design-system-backlog/issues">see a list of all discussions</a></p>
+    {%- endset -%}
+
+    {{- imageCard({
+      src: "/community/images/discuss-and-give-feedback.jpg",
+      alt: "Github issue illustration.",
+      title: "Discuss and give feedback",
+      content: discussAndGiveFeedback
+    }) }}
+  </div>
+  <div class="govuk-grid-column-one-half">
+    {%- set shareYourExamplesAndResearch -%}
+      <p>'<a href="https://design-system.service.gov.uk/community/backlog/">Upcoming components and patterns</a>' shows what we're working on, and how you can help us develop and publish them. Any examples, use cases and research you can share would be very valuable to us.</p>
+    {%- endset -%}
+
+    {{- imageCard({
+      src: "/community/images/share-your-examples-and-research.jpg",
+      alt: "Share your examples and research section illustration.",
+      title: "Share your examples and research",
+      content: shareYourExamplesAndResearch
+    }) }}
+  </div>
+  <div class="govuk-grid-column-one-half">
+    {%- set proposeAChangeToPages -%}
+      <p>Anyone can suggest an improvement, report a bug or correct an error on our pages. Look for the ‘Help improve this page’ section at the end of each page to <a href="https://design-system.service.gov.uk/community/propose-a-content-change-using-github/">propose a change using GitHub</a></p>
+    {%- endset -%}
+
+    {{- imageCard({
+      src: "/community/images/propose-a-change-to-pages.jpg",
+      alt: "Github propose a change illustration.",
+      title: "Propose a change to pages",
+      content: shareYourExamplesAndResearch
```

Comment:
@owenatgov: So @christopherthomasdesign spotted that the "‘[Upcoming components and patterns](https://design-system.service.gov.uk/community/backlog/)‘ shows..." paragraph is duplicated in the preview.

I think this is the culprit due to a faulty copy-paste? From what I can see, there's a content variable that's declared and then called, so it's just a mismatch here.

```suggestion
      content: proposeAChangeToPages
```

Reply:
Would love to see more of the ins and outs of the 'image card' sometime!

Reply:
Leave it with me, I'll be cleaning this up before I go on holiday. I'm also happy to talk through the image card! Sus my calendar 🙂